src/datasource.ts

```diff
@@ -47,18 +47,13 @@ export class DataSource extends DataSourceWithBackend<
           const newFrames = queries
             .filter(query => {
               const queryChannel = replaceWithVars(
-                `ds/${query.datasource?.uid}/${query.path}`,
+                `ds/${query.datasource?.uid}/${query?.path || '.'}`,
```

Comment:
Why the `URL` should end with `.` if no path is informed?

Reply:
Maybe generate a random uuid or number ?

Reply:
How we are using the path passed by the user as the channel identifier (to make sure the will be only one channel opened for that path), if the user inputs an empty path Grafana will not allow empty path in the backend. In these cases we are using a dot path. Just for this cases.